src/data/reducers/loginForm.js

```diff
@@ -0,0 +1,35 @@
+import {
+  LOGIN_FAILURE,
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+} from '../constants/loginForm';
+
+const loginForm = (state = {
+  isAuthenticated: localStorage.getItem('access_token') !== false,
```

Comment:
In Javascript, the `===` (versus `==`) looks for the same type. In this case, your logic is assuming that `localStorage.getItem('access_token')` will return a boolean if access_token does not exist. If `access_token` doesn't exist, it will return `null`. Even though `null` is falsey, it is not explicitly boolean so `localStorage.getItem('access_token') !== false` will actually return `true` even if `access_token` does not exist.

You could use `!!localStorage.getItem('access_token')` instead, which will force the resulting value to be boolean.